batch/batch/driver/instance_collection/pool.py

```diff
@@ -415,7 +415,7 @@ async def user_runnable_jobs(user, remaining):
                     '''
 SELECT job_id, spec, cores_mcpu
 FROM jobs FORCE INDEX(jobs_batch_id_state_always_run_inst_coll_cancelled)
-WHERE batch_id = %s AND state = 'Ready' AND always_run = 1 AND inst_coll = %s
+WHERE jobs.batch_id = %s AND state = 'Ready' AND always_run = 1 AND inst_coll = %s
```

Comment:
```suggestion
WHERE batch_id = %s AND state = 'Ready' AND always_run = 1 AND inst_coll = %s
```

I think this was most of the way reverted but might as well take it 100% so this file doesn't show up in the PR

Reply:
Same for the below query